.changeset/proud-toys-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': patch
```

Comment:
oh and i just realized - this package is on 1.x so it's a `minor` to add this as a new feature so to speak